### PR TITLE
[JENKINS-66729] Un-inlining UsageStatistics/footer.jelly (CSP issue) #Hacktoberfest

### PR DESCRIPTION
**Pull request by abhijeet-chatterjee**
See [JENKINS-66729](https://issues.jenkins-ci.org/browse/JENKINS-66729).
Script Tag for `UsageStatistics/footer.jelly` moved to separate file for CSP reasons.

### Submitter checklist

- [x] (If applicable) Jira issue is well described
- [x] Changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developer, depending on the change). [Examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)
  * Fill-in the `Proposed changelog entries` section only if there are breaking changes or other changes which may require extra steps from users during the upgrade
- [ ] Appropriate autotests or explanation to why this change has no tests
- [ ] For dependency updates: links to external changelogs and, if possible, full diffs

### Desired reviewers

@daniel-beck @Wadeck

### Maintainer checklist

Before the changes are marked as `ready-for-merge`: 

- [ ] There are at least 2 approvals for the pull request and no outstanding requests for change
- [ ] Conversations in the pull request are over OR it is explicit that a reviewer does not block the change
- [ ] Changelog entries in the PR title and/or `Proposed changelog entries` are correct
- [ ] Proper changelog labels are set so that the changelog can be generated automatically
- [ ] If the change needs additional upgrade steps from users, `upgrade-guide-needed` label is set and there is a `Proposed upgrade guidelines` section in the PR title. ([example](https://github.com/jenkinsci/jenkins/pull/4387))
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins-ci.org/issues/?filter=12146)).

### CODE REVIEW

File: core/src/main/resources/hudson/model/UsageStatistics/footer-resources.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Behaviour.addLoadEvent(function() {
+    loadScript("https://usage.jenkins.io/usage-stats.js?${statData}");
```
Comment (daniel-beck) [inline core/src/main/resources/hudson/model/UsageStatistics/footer-resources.js:2]: This won't work. Have you tested it at all?

Comment (MarkEWaite) [reply core/src/main/resources/hudson/model/UsageStatistics/footer-resources.js]: It needs to be tested interactively with a web browser in addition to the test automation.  Human evaluation of presentation of a web page sees things that the automated test assertions may not be checking.

Comment (Wadeck) [reply core/src/main/resources/hudson/model/UsageStatistics/footer-resources.js]: The `statData` is a java variable that is injected in the Jelly. If you are moving to a separate file, this value will be not provided to the JS file.

You can find more information in https://www.jenkins.io/doc/developer/security/xss-prevention/#passing-values-to-javascript.

But yeah, please test it manually ;)

**Comment by abhijeet007rocks8** [comment]
@MarkEWaite @daniel-beck @Wadeck can you review the PR now ?

**Review by Wadeck — APPROVED** [review]
Not manually tested but this version should work a lot better :)

Have you tested it?

**Review by oleg-nenashev — APPROVED** [review]
Yes the new version should work.

We may merge it in 24 hours if there is no negative feedback. Please see the [merge process documentation](https://github.com/jenkinsci/jenkins/blob/master/docs/MAINTAINERS.adoc#merge-process) for more information about the merge process

**Pull request merged**